src/actions/index.js

```diff
@@ -13,7 +15,35 @@ const fetchFormCategories = () => ({
   }
 });
 
-export const loadFormCategories = () => dispatch => dispatch(fetchFormCategories());
+export const loadFormCategories = () => (dispatch, getState) => {
```

Comment:
I think you need to start splitting your actions into separate files, usually it's a good pattern to have one action file per entity, so categoriesActions and formActions for example, to keeps things organized